public/index.html

```diff
@@ -1,19 +1,26 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <!-- Primary Meta Tags -->
+    <title>Estado actual del Coronavirus en REP DOM.</title>
+    <meta name="title" content="Estado actual del Coronavirus en REP DOM.">
     <meta name="description" content="Esta aplicacion busca llevar a sus usuarios informacion resumida y actualizada por el ministerio de Salud Publica sobre la propagacion del virus COVID-19.">
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://www.coronavirus-rd.com/">
     <meta property="og:title" content="Estado actual del Coronavirus en REP DOM.">
     <meta property="og:description" content="Esta aplicacion busca llevar a sus usuarios informacion resumida y actualizada por el ministerio de Salud Publica sobre la propagacion del virus COVID-19.">
-    <meta property="og:image" content="<%= BASE_URL %>logo-mdb-vue-small.png">
-    <meta property="og:url" content="https://www.coronavirus-rd.com/">
-    <meta name="twitter:card" content="summary_large_image">
-    <meta property="og:site_name" content="Healty, Org.">
-    <meta name="twitter:image:alt" content="Corona virus en RD">
+    <meta property="og:image" content="/logo-mdb-vue-small.png">
+
+    <!-- Twitter -->
+    <meta property="twitter:card" content="summary_large_image">
+    <meta property="twitter:url" content="https://www.coronavirus-rd.com/">
+    <meta property="twitter:title" content="Estado actual del Coronavirus en REP DOM.">
+    <meta property="twitter:description" content="Esta aplicacion busca llevar a sus usuarios informacion resumida y actualizada por el ministerio de Salud Publica sobre la propagacion del virus COVID-19.">
+    <meta property="twitter:image" content="/logo-mdb-vue-small.png">
```

Comment:
All of those tag are resuming with this `<meta property="twitter:card" content="summary_large_image">`